Wikipedia/Code/EditorViewController.swift

```diff
@@ -120,6 +140,97 @@ final class EditorViewController: UIViewController, WMFNavigationBarConfiguring
     
     // MARK: - Private Helpers
     
+    private func presentTempEditorSheet() {
+        if let tempUser = authManager.authStateTemporaryUsername {
+            let vm = WMFTempAccountsSheetViewModel(
+                image: "pageMessage",
+                title: WMFLocalizedString("temp-account-edit-sheet-title", value: "You are using a temporary account", comment: "Temporary account sheet for editors"),
+                subtitle: tempEditorSubtitleString(tempUsername: tempUser),
+                ctaTopString: WMFLocalizedString("temp-account-edit-sheet-cta-top", value: "Log in or create an account", comment: "Temporary account sheet for editors, log in/sign up."),
+                ctaBottomString: WMFLocalizedString("temp-account-got-it", value: "Got it", comment: "Got it button"),
+                done: CommonStrings.doneTitle,
+                handleURL: { url in
+                    guard let presentedViewController = self.navigationController?.presentedViewController else {
+                        DDLogError("Unexpected navigation controller state. Skipping Learn More presentation.")
+                        return
+                    }
+                    
+                    let webVC: SinglePageWebViewController
+                    
+                    let config = SinglePageWebViewController.StandardConfig(url: url, useSimpleNavigationBar: true)
+                    webVC = SinglePageWebViewController(configType: .standard(config), theme: self.theme)
+                    
+                    let newNavigationVC = WMFComponentNavigationController(rootViewController: webVC, modalPresentationStyle: .formSheet)
+                    presentedViewController.present(newNavigationVC, animated: true)
+                })
+            let tempAccountsSheetView = WMFTempAccountsSheetView(viewModel: vm)
+            let hostingController = UIHostingController(rootView: tempAccountsSheetView)
+            hostingController.modalPresentationStyle = .pageSheet
+            
+            if let sheet = hostingController.sheetPresentationController {
+                sheet.detents = [.large()]
+            }
+            
+            present(hostingController, animated: true, completion: nil)
+        }
+    }
+    
+    func tempEditorSubtitleString(tempUsername: String) -> String {
+        let openingLink = "<a href=\"\(ipURL)\">"
+        let closingLink = "</a>"
+        let openingBold = "<b>"
+        let closingBold = "</b>"
+        let lineBreaks = "<br/><br/>"
+        let format = WMFLocalizedString("temp-account-edit-sheet-subtitle", value: "Your edit will be attributed to %2$@%1$@%3$@. %4$@IP address%5$@ will be visible to administrators.%6$@If you log in or create an account, your edits will be attributed to a name you choose, among other benefits.",
```

Comment:
New temp name insert is working well. I notice a missing "Your" before the IP address link:

![Screenshot 2025-02-03 at 12 51 13 PM](https://github.com/user-attachments/assets/1aa6709f-0e38-4d08-b55b-6081d70bdfc6)